### PR TITLE
shell_commands: sntp: increase default timeout

### LINKED ISSUE #8473: SNTP not working
#### Description
I'm trying to get NTP time from an SNTP server (Initially it was chrony, then openntpd). At first we tried with our own code in main.c, which was not successful, then switched to the sntp test code in /tests, but that doesn't work, too.

This is on a bluepill board with AT86RF212b transceivers. 6lowpan with RPi (where NTP daemon is) as BR. Tested with public servers, too, same result.

tcpdump shows an incoming NTPv4 packet from the bluepill and an outgoing one from the RPi. Compiled both SNTP and UDP with ENABLE_DEBUG(1), the output follows. All IPv6 addresses are public and routable. 
#### Steps to reproduce the issue
Compile the SNTP test on bluepill board with AT86RF212b transceiver (I suspect the result will be the same with any transceiver board combo). 
#### Expected results
Get ntpdate from server
#### Actual results
```
ntpdate 2001:XXX:XXXX:XXXX::1 100000
udp: GNRC_NETAPI_MSG_TYPE_SND
Error receiving message
Error in synchronization
> udp: GNRC_NETAPI_MSG_TYPE_RCV
udp: unable to forward packet as no one is interested in it
```
#### Versions
RIOT from trunk. main(): This is RIOT! (Version: 2018.04-devel-82-g0abbd-x220)
Compiled on Linux (Arch x86_64), gcc-arm-none-eabi-bin 7_2017_q4_major-1

tcpdump output:

```
20:40:40.641793 IP6 2001:XXX:XXXX:XXXX:7270:aa52:4269:ea76.49186 > 2001:XXX:XXXX:1044::1.123: NTPv4, Client, length 48

20:40:40.642868 IP6 2001:XXX:XXXX:1044::1.123 > 2001:XXX:XXXX:XXXX:7270:aa52:4269:ea76.49186: NTPv4, Server, length 48
```

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
This PR

* specifies the time unit used for the timeout with the `ntpdate` command
* increases the default timeout as 5000μs is far to small

### Issues/PRs references
Fixes #8473

### CODE REVIEW

File: sys/shell/commands/sc_sntp.c [hunk 1]
```diff
@@ -27,12 +27,12 @@
...
 
-#define _DEFAULT_TIMEOUT (5000U);
+#define _DEFAULT_TIMEOUT (500000LU)
 
 static void _usage(char *cmd)
 {
-    printf("Usage: %s <server addr>[%%<interface>] [<timeout>]\n", cmd);
-    puts("default: timeout = 5000");
+    printf("Usage: %s <server addr>[%%<interface>] [<timeout in us>]\n", cmd);
+    printf("default: timeout = %lu", _DEFAULT_TIMEOUT);
```
Comment (bergzand) [inline sys/shell/commands/sc_sntp.c]: Missing newline at the end of the printf here.

Comment (miri64) [reply sys/shell/commands/sc_sntp.c]: Added

**Comment by bergzand** [comment]
Tested, timeout worked as expected. Please squash.

**Comment by miri64** [comment]
Squashed

**Pull request merged**